app/components/modules/UserWallet.jsx

```diff
@@ -87,19 +87,36 @@ class UserWallet extends React.Component {
             })
         }
 
+        // Sum conversions
+        let conversionValue = 0;
+        const conversions = account.get('other_history').filter(a => {
+            return a.getIn([1, 'op', 0]) === 'convert';
+        }).map(c => {
+            const timestamp = new Date(c.getIn([1, 'timestamp'])).getTime();
+            const finishTime = timestamp + (86400000 * (timestamp <= 1481040000000 ? 7 : 3.5)); // add conversion delay before/after hardfork change
```

Comment:
please pull logic like this out into appropriately-named functions so that someone reading this code gets a semantic hint (i know what it does today, but in 8 months i will absolutely not remember the specifics of HF16).